homeassistant/components/tuya/cover.py

```diff
@@ -117,7 +117,7 @@ class TuyaCoverEntityDescription(CoverEntityDescription):
     "jdcljqr": (
         TuyaCoverEntityDescription(
             key=DPCode.CONTROL,
-            current_position=DPCode.PERCENT_STATE,
+            current_position=DPCode.PERCENT_CONTROL,
```

Comment:
Are you correcting a typo or do you have a different variant?

Reply:
I'm suppose this should be a copy-paste error.
I own here a tuya robot curtain. But see comment below to let me know how to test them out